src/vega.schema.ts

```diff
@@ -1,4 +1,5 @@
 import {AggregateOp} from 'vega';
+import {FoldTransform as VgFoldTransform} from 'vega-typings';
```

Comment:
Great!  This is exactly what we need.  

@domoritz  I wonder if we should import and reexport from this file so the rest of the codebase do not have to do `as` and thus will have consistent name?